### PR TITLE
Add deploy library action

### PR DESCRIPTION
**Pull request by brolewis**
First time writing a full action, so I probably screwed something up...

### CODE REVIEW

File: deploy-library/action.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: deploy-library
+description: "Deploy a library"
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (brolewis) [reply deploy-library/action.yml:6]: https://www.youtube.com/watch?v=WlVsrPrRCE8

[the remark this answers is not in the document]
Comment (brolewis) [reply deploy-library/action.yml:6]: Actually, it doesn't: https://github.com/unitasglobal/nio-shared-actions/blob/main/python-setup/action.yml

[the remark this answers is not in the document]
Comment (cmahanna-ug) [reply deploy-library/action.yml:6]: https://www.youtube.com/watch?v=dQw4w9WgXcQ

[the remark this answers is not in the document]
Comment (brolewis) [reply deploy-library/action.yml:6]: (But every other action _calls_ `python-setup` so maybe that should be the one point of calling `checkout`?)

**Review by cmahanna-ug — APPROVED** [review]

**Pull request merged**